BadApi/Controllers/AccountController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using BadApi.Controllers.Base;
+using BadApi.Domain;
+using BadApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+
+using Dapper;
+using BadApi.Models.Account;
+
+namespace BadApi.Controllers
+{
+	[Authorize]
+	[RoutePrefix("api/account")]
+    public class AccountController : BaseApiController
+    {
+        // GET: Account
+		[AllowAnonymous]
+		[HttpGet]
+		[Route("login")]
+        public async Task<Account> Login([FromUri] LoginModel model)
+        {
+            var connection = GetConnection();
```

Comment:
Consider avoiding having to instantiate dependencies within the code. Use dependency injection instead as it will make this class more testable and loosely coupled with GetConnection() method.